app/models/concerns/waste_carriers_engine/can_have_payment_type.rb

```diff
@@ -42,11 +42,11 @@ def bank_transfer?
         payment_type == BANKTRANSFER
       end
 
-      def worldpay?
+      def online?
```

Comment:
This gets tricky because when the conditional method is called `worldpay?`, it reflects the `payment_type`. Could this be misleading for the next people?